source/Hit:Stand.py

```diff
@@ -0,0 +1,17 @@
+def Hit_Stand(self, Player):
+  def_input = input("Would you like to Hit or Stand?  ")
+    while def_input not in ["Hit", "Stand"]:
```

Comment:
The code is improperly indented.

Reply:
Im assuming the while loop is indented too far?

Reply:
line 3 is more indented than line 2 this causes an error for me